src/main/java/org/kiwiproject/eureka/junit/EurekaServerExtension.java

```diff
@@ -76,4 +79,47 @@ public void afterAll(ExtensionContext context) {
         EurekaTestHelpers.resetStatsMonitor();
     }
 
+    /**
+     * Helper method to access {@link EmbeddedEurekaServer#getRegistry()}'s {@code clearRegisteredApps()}.
+     */
+    public void clearRegisteredApps() {
+        eurekaServer.getRegistry().clearRegisteredApps();
+    }
+
+    /**
+     * Helper method to access {@link EmbeddedEurekaServer#getRegistry()}'s
+     * {@code registerApplication(appName, instanceId, vipAddress, status)}.
+     */
+    public void registerApplication(String appName, String instanceId, String vipAddress, String status) {
+        eurekaServer.getRegistry().registerApplication(appName, instanceId, vipAddress, status);
+    }
+
+    /**
+     * Helper method to access {@link EmbeddedEurekaServer#getRegistry()}'s {@code registeredApplications()}.
+     */
+    public List<Application> registeredApplications() {
```

Comment:
Is there a reason this isn't a "getter" like the one below? i.e. why isn't it `getRegisteredApplications()`?

Reply:
That is just what we had down below, I can change it

Reply:
I changed it

Reply:
Do you want the underlying changed too?  This will break the API that is being used today?

Reply:
OK I didn't see the previous stuff since GitHub only showed the diffs. So, maybe change it back?

Reply:
And would changing the `getRegisteredApplication` to just `registeredApplication` make sense? Would that also break an API?

Reply:
So making any of these "new" helper methods different names won't break anything.  If you want the name to change inside of `EmbeddedEurekaServer` then that could/will break the API since they are all accessed right now through `EUREKA.getEurekaServer().getRegistry().xxx`

So you tell me how you want the helper methods to look and I'll make it so

Reply:
I don't think it matters all that much. Let's go with _not_ breaking existing public APIs when the change doesn't add a lot of value. If you think it _does_ add value, change it, otherwise don't.

Reply:
Ok, I'll leave the existing API the same, but the helper methods now have `get` in front of them.  This PR is done then.